### ISSUE TITLE
ImportError?

### ISSUE THREAD

**Issue by shackshack** [InstaPy/InstaPy#5100]
>>> 
 RESTART: C:\Users\shack\AppData\Local\Programs\Python\Python37-32\Lib\site-packages\instapy\like_atm.py 
Traceback (most recent call last):
  File "C:\Users\shack\AppData\Local\Programs\Python\Python37-32\Lib\site-packages\instapy\like_atm.py", line 8, in <module>
    from instapy import InstaPy
  File "C:\Users\shack\AppData\Local\Programs\Python\Python37-32\Lib\site-packages\instapy\instapy.py", line 27, in <module>
    from . import __version__
ImportError: attempted relative import with no known parent package
>>> 

Any idea what I should do for this?

**Comment by napuzba** [comment]
See [ImportError: attempted relative import with no known parent package](https://napuzba.com/a/import-error-relative-no-parent)

**Comment by shackshack** [comment]
> See [ImportError: attempted relative import with no known parent package](https://napuzba.com/a/import-error-relative-no-parent)

Thanks, I am still a little confused because the folder structure they are referencing is different than mine and I am not sure what to include or omit.
They say:
project
 ├── config.py
 └── package
     ├── __init__.py
     └── demo.py

But mine is:
Python\Python37-32\Lib\site-packages\instapy

I am not sure what would be considered the "project", is it the .py file I am trying to run? And would I do this right above "site-packages"?

I tried creating a "new_project" folder within Lib. Added the .py files they suggested and tried to run it again and still nothing
